packages/manager/apps/web/client/app/hosting/multisite/update/hosting-multisite-update.html

```diff
@@ -58,6 +64,7 @@
                     runtime="selected.runtime"
                     runtimes="model.runtimes"
                     ssl="selected.domain.ssl"
+                    can-modify-domain-with-git="canModifyDomainWithGit"
```

Comment:
Why binding the whole function ? The result as a boolean should be enough
```suggestion
                    can-modify-domain-with-git="canModifyDomainWithGit()"
```